### DISCUSSION TITLE
偶现pag动画花屏

### DISCUSSION THREAD

**Discussion by lyquanci** [Tencent/libpag#2144] [Q&A]
![image](https://github.com/Tencent/libpag/assets/160443784/7a8e6521-95a8-497b-adc4-109194ddbf39)

**Comment by kevingpqi123** [comment]
这类问题建议提供完整的复现 demo 及复现机型，仅仅靠描述难以定位问题

**Comment by CodeJhF** [comment]
之前请你补充的复现信息（demo / 版本号 / 机型 / 日志）一直没有收到，缺少这些信息无法继续定位，先关闭本条。
如果问题还在，欢迎补齐上述信息后回复或重开，我们会优先跟进。（stale-close）
